tests/e2e/multicluster/multicluster_primaryremote_test.go

```diff
@@ -0,0 +1,353 @@
+
+						// Run the curl command from the sleep pod in the Primary Cluster and get response list to validate that we get responses from both clusters
+						primaryResponses := strings.Join(getListCurlResponses(sleepPodNamePrimary, kubeconfig), "\n")
+						Expect(primaryResponses).To(ContainSubstring("Hello version: v1"), "Responses from Primary Cluster are not the expected")
+						Expect(primaryResponses).To(ContainSubstring("Hello version: v2"), "Responses from Primary Cluster are not the expected")
+						Success("Sample app is accessible from both clusters")
+					})
+				})
+
+				When("sample apps are deleted in both clusters", func() {
+					BeforeAll(func(ctx SpecContext) {
+						// Delete the entire sample namespace in both clusters
+						Expect(kubectl.DeleteNamespace("sample", kubeconfig)).To(Succeed(), "Namespace failed to be deleted on Primary Cluster")
+						Expect(kubectl.DeleteNamespace("sample", kubeconfig2)).To(Succeed(), "Namespace failed to be deleted on Remote Cluster")
+					})
+
+					It("sample app is deleted in both clusters and the namespace is empty", func(ctx SpecContext) {
+						common.CheckNamespaceEmpty(ctx, clPrimary, "sample")
+						common.CheckNamespaceEmpty(ctx, clRemote, "sample")
+						Success("Sample app is deleted in both clusters")
+					})
+				})
+
+				When("control plane namespace and gateway are deleted in both clusters", func() {
+					BeforeEach(func() {
+						// Delete the Istio CR in both clusters
+						Expect(kubectl.Delete(controlPlaneNamespace, "istio", istioName, kubeconfig)).To(Succeed(), "Istio CR failed to be deleted")
+						Expect(kubectl.Delete(controlPlaneNamespace, "remoteistio", istioName, kubeconfig2)).To(Succeed(), "RemoteIstio CR failed to be deleted")
+						Success("Istio CR is deleted in both clusters")
+
+						// Delete the gateway in both clusters
+						Expect(kubectl.DeleteFromFile(eastGatewayYAML, kubeconfig)).To(Succeed(), "Gateway deletion failed on Primary Cluster")
+						Expect(kubectl.DeleteFromFile(westGatewayYAML, kubeconfig2)).To(Succeed(), "Gateway deletion failed on Remote Cluster")
+
+						// Delete the namespace in both clusters
+						Expect(kubectl.DeleteNamespace(controlPlaneNamespace, kubeconfig)).To(Succeed(), "Namespace failed to be deleted on Primary Cluster")
+						Expect(kubectl.DeleteNamespace(controlPlaneNamespace, kubeconfig2)).To(Succeed(), "Namespace failed to be deleted on Remote Cluster")
+					})
+
+					It("removes everything from the namespace", func(ctx SpecContext) {
+						Eventually(clPrimary.Get).WithArguments(ctx, kube.Key("istiod", controlPlaneNamespace), &appsv1.Deployment{}).
+							Should(ReturnNotFoundError(), "Istiod should not exist anymore")
+						common.CheckNamespaceEmpty(ctx, clPrimary, controlPlaneNamespace)
+						common.CheckNamespaceEmpty(ctx, clRemote, controlPlaneNamespace)
+						Success("Namespace is empty")
+					})
```

Comment:
Same with this?

Reply:
Done, I only leave there the validation to check that if we delete the Istio resource, the istiod pod is deleted